packages/opentelemetry-sdk-trace-base/test/common/export/SimpleSpanProcessor.test.ts

```diff
@@ -36,7 +36,8 @@ import {
 } from '../../../src';
 import { TestStackContextManager } from './TestStackContextManager';
 import { TestTracingSpanExporter } from './TestTracingSpanExporter';
-import { Resource, ResourceAttributes } from '@opentelemetry/resources';
+import { Attributes } from '@opentelemetry/api';
```

Comment:
note for reviewer: leftover from https://github.com/open-telemetry/opentelemetry-js/pull/5009